plugins/woocommerce/src/Internal/Admin/ProductReviews/ReviewsListTable.php

```diff
@@ -95,6 +95,7 @@ public function prepare_items() : void {
 		// Include the offset argument.
 		$args = wp_parse_args( $this->get_offset_arguments(), $args );
 
+		$args = apply_filters( 'woocommerce_product_reviews_comment_args' ,$args );
```

Comment:
I'd suggest a slightly different hook name, which though longer than the one you provided makes it clear that this specifically relates to the admin list table (and not to the args of product review comment queries in general):

```suggestion
		/**
		 * Provides an opportunity to alter the comment query arguments used within
		 * the product reviews admin list table.
		 *
		 * @param array $args Comment query args.
		 */
		$args = (array) apply_filters( 'woocommerce_product_reviews_list_table_prepare_items_args', $args );
```

(Also added a docblock and tidied formatting.)